config/config.go

```diff
@@ -28,7 +28,6 @@ type Driver struct {
 	Token        string        `json:"token,omitempty"`
 	URL          string        `json:"url,omitempty"`
 	Protocol     string        `json:"protocol,omitempty"`
-	Project      string        `json:"project,omitempty"`
```

Comment:
What is the reason for this change?

Reply:
Because driver is not associated with project anymore. Driver can create new project and it takes project name as argument for appropriate methods.

Reply:
The client object should still be associated with a project I think because except for a few operations all the operations are within the scope of a single project. Can project creation/deletion functions be outside the close object?
From a user perspective, a regular user would always pass project when instantiating the client object so that all operations are properly scoped. Which is why it is better to have this in the config

Reply:
ok - higher level client (Client interface from tigris package) is now associated with project.

Reply:
Would this still be in the config?